components/LessonCard.tsx

```diff
@@ -45,14 +46,14 @@ const ReviewButton: React.FC<ReviewButtonProps> = props => {
     return null
   }
 
-  const pendingSubmissionsCount =
-    data &&
-    data.submissions.reduce((acc, val) => {
-      if (val.status === SubmissionStatus.Open) {
-        return acc + 1
-      }
-      return acc
-    }, 0)
+  const submissions = get(data, 'submissions') || []
```

Comment:
We dont have to do `|| []` ourselves, lodash provides third parameter for that use case

const submissions = get(data, 'submissions', [])